### PR TITLE
Package fred.0.1.0

### PR DESCRIPTION
**Pull request by gborough**
### `fred.0.1.0`
API binding for Federal Reserve Economic Data
API binding for Federal Reserve Economic Data

---
* Homepage: https://github.com/gborough/fred
* Source repo: git+https://github.com/gborough/fred.git
* Bug tracker: https://github.com/gborough/fred/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

**Comment by gborough** [comment]
Not too sure why the tests are failing, I tried locally reproducing them and they were passing. They depend on fetching external data via curl so I suspect something went wrong there inside the pipeline.

Strictly speaking these tests are not mandatory but mere sanity checks, so I am ok with releasing this. I can also disable tests altogether if that helps.

**Comment by samoht** [comment]
Are you running those tests on Windows by any chance? Those kinds of diffs are pretty weird and might indicate something happening with newlines characters:

```
# -  print_endline o ; [%expect {| t |}]
# +  print_endline o ; [%expect {| f |}]
```

I'll disable the tests and merge. Thanks for your submission!

File: packages/fred/fred.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+license: "MIT"
+tags: ["api binding" "federal reserve" "ecomonic data"]
+homepage: "https://github.com/gborough/fred"
+doc: "https://gborough.github.io/fred/fred"
+bug-reports: "https://github.com/gborough/fred/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "dune" {>= "2.7"}
+  "ezcurl-lwt"
+  "ppx_expect" {with-test}
```
Comment (samoht) [inline packages/fred/fred.0.1.0/opam:16]:
```suggestion
```

File: packages/fred/fred.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```
Comment (samoht) [inline packages/fred/fred.0.1.0/opam:32]:
```suggestion
```

**Pull request merged**